operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -264,6 +264,11 @@ rbac:
 serviceMonitor:
   # -- If enabled, ServiceMonitor resources for Prometheus Operator are created
   enabled: false
+  clusterLabel:
+    # -- Whether to include 'cluster' label in metrics
+    enabled: true
```

Comment:
This turned out to be cleaner code as apparently deciding whether a value is "null" or "" empty string is a hassle in Helm.

Reply:
I don't think we should be sacrificing the interface of the chart for implementation details. Have you tried the helm `empty` function? It should return `false` on both `''` and `null`

Reply:
That is the problem, I want to differentiate between "" and null. And I think it's actually more clear to have a separate `enabled` value, since that semantics is easier to understand then the 3-way choice other wise, compare:

value - If null then disabled , if empty then set to default, otherwise set to the value

VS

enabled - whether cluster label is generated or not
value - empty means the default, otherwise what you write here

Reply:
oh 🤦 I didn't realize there are two variables (enabled & useDefault), not only one (enabled). I'm ok to leave as-is then.

If maybe other folks involved have opinions, I'm interested to hear those, so can we leave this discussion open?

Reply:
I talked to @osg-grafana  and having a single value encode the 3 choices is fine.
Also I realized that we are using `enabled` keyword for much larger things like whole services and features, so I think it would dilute the meaning of `enabled` if we used it on such low level. Anyway, added a commit to go to single value.